pages/search.tsx

```diff
@@ -11,13 +28,148 @@ export const getServerSideProps: GetServerSideProps = async ({ query }) => ({
+      'group focus:outline-none focus:ring-2 focus:ring-gray-800',
+      'focus:ring-gray-800 focus:z-10 relative'
+    )}
+    style={{ width: 'calc(100% + 32px)' }}
+    onClick={onClick}
+  >
+    <span
+      className={classNames(
+        'rounded-full w-4 h-4 border-4 border-white shadow-md',
+        'transform -translate-y-2'
+      )}
+      style={{
+        backgroundColor: POI_DATA.fillColorMap.get(properties.category),
+      }}
+    />
+    <div
+      className={classNames(
+        'pb-2 border-b border-gray-300 flex-grow border-dashed',
+        'group-hover:border-opacity-0 group-focus:border-opacity-0 transition'
+      )}
+    >
+      <h6 className="text-sm leading-4">{properties.name}</h6>
+      <div className="flex gap-x-2 flex-wrap">
+        <span className="text-sm text-gray-500">{properties.category}</span>
+        <span className="text-sm text-gray-400 italic">
+          (&thinsp;{properties.conditions}&thinsp;)
+        </span>
+      </div>
+    </div>
+  </button>
+)
+
+const SearchResultItem: FC<SearchResultItemPropType> = ({
+  name,
+  searchTerm,
+  onClick,
+}) => (
+  <button
+    onClick={onClick}
+    className={classNames(
+      'flex gap-2 items-center pt-3 text-left w-full',
+      'hover:bg-gray-100 rounded transition px-4 -ml-4',
+      'group focus:outline-none focus:ring-2 focus:ring-gray-800',
+      'focus:ring-gray-800 focus:z-10 relative'
+    )}
+    style={{ width: 'calc(100% + 32px)' }}
+  >
+    <span className="text-gray-300 transform -translate-y-2">
+      <GeoPinIcon />
+    </span>
+    <div
+      className={classNames(
+        'pb-2 border-b border-gray-300 flex-grow border-dashed',
+        'group-hover:border-opacity-0 group-focus:border-opacity-0 transition'
+      )}
+    >
+      <h6 className="text-sm leading-4">
+        {name.split(searchTerm)[0]}
+        <span className="text-gray-400 text-light">{searchTerm}</span>
+        {name.split(searchTerm)[1]}
+      </h6>
+    </div>
+  </button>
 )
 
+export const Search: FC = () => {
+  const [inputVal, setInpuval] = useState('')
+  const { pathname, push, query } = useRouter()
+  const mappedQuery = mapRawQueryToState(query)
+  const { results } = useGeocodedPlace(inputVal)
+
+  const clickHandler = useCallback(
+    (coordinates: [latitude: number, longitude: number]) => {
+      const nextQuery = {
+        ...mappedQuery,
+        latitude: coordinates[1],
+        longitude: coordinates[0],
+        zoom: 14,
+      }
+      void push({ pathname, query: nextQuery }, undefined, { shallow: true })
+    },
+    [push, pathname, mappedQuery]
+  )
+
+  return (
+    <div>
+      <h4 className="font-bold text-xl hidden sm:block">Standort</h4>
+      <p className="text-sm text-gray-500 hidden sm:block">
+        Finde deinen Bezirk, deine Straße oder einen anderen Ort in Berlin.
+      </p>
+
+      <input
+        type="text"
+        placeholder="Geben Sie einen Ort an"
```

Comment:
Let's make this less formal: "Gib hier einen Ort an"

Reply:
Will do!